### PR TITLE
Fixed race condition in telemetry test.

### PR DESCRIPTION
**Pull request by brandonforster**
Issue #1191 

Signed-off-by: Brandon Forster <brandonforster@gmail.com>

### CODE REVIEW

**Comment by brandonforster** [comment]
recheck

**Comment by brandonforster** [comment]
recheck

**Comment by MightyNerdEric** [comment]
recheck

File: internal/pkg/telemetry/telemetry.go [hunk 1]
```diff
@@ -43,6 +43,7 @@ type CpuUsage struct {
 var once sync.Once
 var lastSample CpuUsage
 var usageAvg float64
+var WaitGroup sync.WaitGroup
```
Comment (tsconn23) [inline internal/pkg/telemetry/telemetry.go]: This var doesn't need to be exported. Should be `waitGroup`. Otherwise LGTM.

Comment (brandonforster) [reply internal/pkg/telemetry/telemetry.go]: Addressed.

**Pull request merged**